src/litserve/utils.py

```diff
@@ -98,20 +98,32 @@ class WorkerSetupStatus:
 
 
 def configure_logging(
-    level=logging.INFO, format="%(asctime)s - %(name)s - %(levelname)s - %(message)s", stream=sys.stdout
+    level=logging.INFO, format="%(asctime)s - %(name)s - %(levelname)s - %(message)s", stream=sys.stdout, use_rich=False
 ):
     """Configure logging for the entire library with sensible defaults.
 
     Args:
         level (int): Logging level (default: logging.INFO)
         format (str): Log message format string
         stream (file-like): Output stream for logs
+        use_rich (bool): Whether to use rich for logging
 
     """
-    # Create a library-wide handler
-    handler = logging.StreamHandler(stream)
-
-    # Set formatter with user-configurable format
+    if use_rich:
+        try:
+            from rich.logging import RichHandler
+            from rich.traceback import install
+
+            install(show_locals=True)
+            handler = RichHandler(rich_tracebacks=True, show_time=True, show_path=True)
+        except ImportError:
+            logger.warning("Rich is not installed, using default logging")
+            handler = logging.StreamHandler(stream)
+            formatter = logging.Formatter(format)
+            handler.setFormatter(formatter)
```

Comment:
is this the default?

Reply:
yes, by default we use the builtin steaming handler